pkg/config/autodiscover/csi_info.go

```diff
@@ -0,0 +1,37 @@
+package autodiscover
+
+import (
+	"fmt"
+	"strings"
+)
+
+// PodSetList holds the data from an `oc get deployment/statefulset -o json` command
+
+// PodSetResource defines deployment/statefulset resources
+
+func getpackageandorg(csi string) (packag, organization string) {
+	orgpack := fmt.Sprintf("./get-csi-info.sh %s", csi)
+	out := execCommandOutput(orgpack)
+	packag = ""
+	organization = ""
+	if out != "" {
+		out := strings.Split(out, " ")
+		packag = out[0]
+		organization = out[1]
+	}
+	return packag, organization
+
+}
+
+// GetTargetCsi will return all podsets(deployments/statefulset )that have pods with a given label.
+func GetTargetCsi() ([]string, error) {
+	ocCmd := "oc get csidriver -o go-template='{{ range .items}}{{.metadata.name}} {{end}}'"
```

Comment:
We already have a csidriver handler, which is being used in the diagnostic test suite. It might be a good idea to reuse it.

Reply:
agreed to keep it with this function

Reply:
Reusing the handler would result in a lot more code, and since it's only used in the diagnostic test suite we can't reuse the output at the moment. Once the suite is moved to a set of helper functions, we could use it.